io.github.anirbandey1.ChatGPTElectron.yml

```diff
@@ -0,0 +1,60 @@
+app-id: io.github.anirbandey1.ChatGPTElectron
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "22.08"
+command: run.sh
+rename-desktop-file: chatgpt-electron.desktop
+rename-icon: chatgpt-electron
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=dri
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+modules:
+
+  - name: run
```

Comment:
this should be in the main module 'chatgtp-electrion'

Reply:
ok doing it